[Term Entry] Python String Function: isspace() (#3322)
Codecademy/docs: content/python/concepts/strings/terms/isspace/isspace.md

---
Title: '.isspace()'
Description: 'Checks if all the characters in a string are whitespace characters.'
Subjects:
  - 'Data Science'
  - 'Computer Science'
Tags:
  - 'Strings'
  - 'Methods'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/analyze-data-with-python'
  - 'paths/data-science'
---

> The `.isspace()` string method takes in a string and returns `True` if all the characters in a string are whitespaces, otherwise False.

Reviewer: ```suggestion
The **`.isspace()`** string method takes in a string and returns `True` if all the characters in a string are whitespaces, otherwise `False`.
```

> **`Whitespace characters include:`**

Reviewer: ```suggestion
Whitespace characters include:
```

- Space (' ')
- Horizontal tab ('\t')
- Newline ('\n')
- Vertical tab ('\v')

## Syntax

```py
string.isspace()
```

> 

Reviewer: ass a line after the syntax example explaining what `space` is.

---
Title: '.isspace()'
Description: 'Checks if all the characters in a string are whitespace characters.'
Subjects:
  - 'Data Science'
  - 'Computer Science'
Tags:
  - 'Strings'
  - 'Methods'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/analyze-data-with-python'

>   - 'paths/data-science'

Reviewer 2: ```suggestion
```

Reviewer 2: there should only be two items under catalog content

---

> The **`.isspace()`** string method takes in a string and returns `True` if all the characters in a string are whitespaces, otherwise `False`.

Reviewer 2: ```suggestion
The **`.isspace()`** string method takes in a string and returns `True` if the entire string is composed of whitespace characters, otherwise `False`.
```

Whitespace characters include:

- Space (' ')
- Horizontal tab ('\t')
- Newline ('\n')
- Vertical tab ('\v')

## Syntax

```pseudo

> string.isspace()

Reviewer 2: ```suggestion
some_str.isspace()
```

```

> `space` is the string being checked for whitespace.

Reviewer 2: ```suggestion
`some_str` is the string being checked for whitespace.
```

## Example

> The example below uses `.isspace()` to check if all characters in the string `txt` are whitespace characters:

Reviewer 2: ```suggestion
The example below uses `.isspace()` to evaluate the string `txt`:
```

```py
txt = "   Welcome   "
x = txt.isspace()
print(x)
```

The output will look like this:

```shell
False
```

## Codebyte Example

> The following example returns `True` because all the characters in the string `my_string` are whitespace characters:

Reviewer 2: ```suggestion
The following example is runnable and checks if `my_string` consists entirely of whitespace characters:
```
